shared/data/src/commonMain/graphql/operations.graphql

```diff
@@ -69,6 +69,7 @@ query GetPartnerGroups {
         partners {
             name
             logoUrl
+            logoUrlDark
```

Comment:
Does that work? I would have expected something like

```
            logoUrl
            logoUrlDark: logo(dark: true)
```

It's a new GraphQL argument, not field

Reply:
Yes it seems to work 🤷‍♀️ I will change this anyway

Reply:
Fun stuff, it worked because of reflection. See https://github.com/joreilly/Confetti/pull/1233.